cmd/kube-proxy/app/server.go

```diff
@@ -93,20 +97,45 @@ func (s *ProxyServer) Run(_ []string) error {
 	serviceConfig := config.NewServiceConfig()
 	endpointsConfig := config.NewEndpointsConfig()
 
-	protocol := iptables.ProtocolIpv4
-	if s.BindAddress.To4() == nil {
-		protocol = iptables.ProtocolIpv6
+	protocol := utiliptables.ProtocolIpv4
+	if net.IP(s.BindAddress).To4() == nil {
```

Comment:
Probably from the last merge fix on my part. That was this file, reviewing it again for anything else I might have mis-merged.